integrations/sonarqube/tests/test_sync.py

```diff
@@ -0,0 +1,57 @@
+# from typing import Any
```

Comment:
what is this for?

Reply:
This is the sync code that kept failing here and on Jira that I couldn't resolve. We agreed to take it out for now.